core/src/main/java/io/kestra/scheduler/events/TriggerExecuted.java

```diff
@@ -0,0 +1,22 @@
+package io.kestra.scheduler.events;
+
+import io.kestra.core.models.executions.Execution;
+import io.kestra.core.models.triggers.TriggerId;
+
+import java.time.Instant;
+
+/**
+ * A trigger was executed.
+ */
+public record TriggerExecuted(
```

Comment:
```suggestion
public record TriggerEvaluated(
```

Or TriggerEvaluation but we use evaluate not execute so it didn't conflict with the execution itself which is not yet started